cantabular/contract.go

```diff
@@ -71,7 +71,7 @@ type SearchDimensionsRequest struct {
 // POST [cantabular-ext]/graphql
 // with a query to obtain variables
 type GetDimensionsResponse struct {
-	Dataset gql.DatasetVariables `json:"dataset"`
+	Dataset gql.Dataset `json:"dataset"`
```

Comment:
Was this incorrect?

Reply:
It was needlessly split into two different structs, where in reality it's the same struct that can have one of two fields returned